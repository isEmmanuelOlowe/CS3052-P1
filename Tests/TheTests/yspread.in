200
0.840485369 3964647.74
0.446583435 3533360.97
0.886428433 3186927.72
0.58409022 155828.494
0.383715875 1593686.27
0.0588589136 6910043.73
0.163545951 8998543.06
0.533064714 1590715.03
0.582699021 6041441.9
0.390478195 2699711.18
0.742377406 2934005.7
0.0755380785 2985256.06
0.857377943 4049826.34
0.66283066 9419683.23
0.00275508143 8464757.8
0.532596024 4623792.45
0.265612235 7878766.21
0.306785131 9827522.63
0.608715653 6008551.36
0.885895131 2124387.98
0.151859864 3046571.02
0.387476951 3376619.03
0.753553276 6436098.29
0.531628252 6036160.99
0.652488447 4593603.16
0.946370486 3271811.64
0.943890339 3680398.67
0.51659995 74282.6172
0.0242991556 2727709.53
0.20496351 5919545.02
0.0593686934 8776933.5
0.302829184 2608425.52
0.498198059 8914952.2
0.286413994 7100255.81
0.675540671 8649235.77
0.959635562 4584899.73
0.376551281 7746754.06
0.354533877 2286391.16
0.669765832 3003182.48
0.565954509 7189665.72
0.39061191 8244653.13
0.84400846 8187663.11
0.943395886 1804677.7
0.520665778 4248867.65
0.913508169 656437.549
0.761364127 8825845.73
0.688256842 3989225.46
0.405008799 7615483.04
0.484633905 1252511.38
0.873121166 2224625.53
0.001413962 5288218.98
0.0186973976 8605138.13
0.242884021 8148919.42
0.965732518 3145718.44
0.809431695 9355601.05
0.220135508 4921091.53
0.289029058 5763537.38
0.261323339 3210669.76
0.00181729961 1739878.58
0.241174551 447841.404
0.701624649 4154511.88
0.503910344 2218456.91
0.393063444 670302.114
0.218142211 4794765.27
0.916203193 2195112.85
0.192693937 3502217.51
0.633681628 2112349.79
0.783411413 535653.932
0.4440966 306739.171
0.932180218 1764128.88
0.472844835 9096482.73
0.695567358 8716954.61
0.455090556 9301896.21
0.893304001 3985667.82
0.838734383 6935467.47
0.651269549 7396991.36
0.577212317 6781542.4
0.935388054 2731182.81
0.0474125134 6619388.85
0.618194778 3730383.9
0.377305582 1486279.86
0.0256878246 6445914.01
0.0770188061 8413805.13
0.255986141 7426339.17
0.377647585 9018442.3
0.211434985 3195299.25
0.251315083 6485325
0.250922181 2286443.17
0.136715085 9432211.62
0.548707187 2700607.67
0.865080021 3240214.47
0.680059336 2967033.48
0.876307513 8331472.54
0.0731154782 6496645.58
0.253581256 8985469.69
0.841889925 6113186.28
0.372587091 8320193.01
0.109214906 7570482.23
0.559329326 8508119.76
0.343096982 8575804.62
0.345197295 6917696.04
0.958707883 8935846.53
0.981544357 1217351.95
0.614776279 550247.948
0.376472598 381266.756
0.281923955 5256048.56
0.607242966 5605379.61
0.446534988 8164395.85
0.471854014 271116.801
0.292723682 2848983.05
0.0175794171 1956801.44
0.573165442 8295992.31
0.732976481 1047806.02
0.223888398 1190975.2
0.739051619 9472528.62
0.826131565 8213569.75
0.256495729 2509799.72
0.388221673 3381931.97
0.266409127 5273184.76
0.870894566 4012205.32
0.294573819 458917.929
0.560002386 3942465.63
0.822576867 3108211.41
0.0911435897 4754814.9
0.917044094 2624010.93
0.332093346 9784358.99
0.240634874 9020740.35
0.752286033 3731643.72
0.900627876 4578273.88
0.565923483 4228945.12
0.353870876 9704621.57
0.179091359 4315065.15
0.337176765 2153191.13
0.0447257049 4543677.53
0.0621491404 6838824.54
0.316171237 5513752.83
0.500765995 2682179.05
0.964826539 625369.632
0.806315558 7295757.91
0.705017961 1519744.11
0.842565842 7299574.46
0.955447777 6179285.83
0.527085486 4793881.26
0.607612115 293865.903
0.592769648 475693.664
0.904394395 4026469.3
0.310837454 3301310.69
0.734149534 8549447.44
0.340458593 986265.335
0.985167064 4017394.94
0.831962391 6295947.81
0.79122669 4371197.3
0.1351349 594618.604
0.930451652 5438238.99
0.389894587 8848990.19
0.85686265 4145575.16
0.107756722 2867585.84
0.777602184 5766759.95
0.808677186 1422509.31
0.410024864 7480942.22
0.344459535 4751157.67
0.0236477474 4167988.46
0.717891538 5258110.1
0.694865075 2360226.79
0.762750408 7799506.06
0.370517959 1215627.16
0.766006338 9282514.33
0.983660708 2895795.33
0.0378130858 4281069.16
0.0155956066 5983264.48
0.130679 7567238.43
0.217496166 3745671.64
0.205663394 5777970.01
0.748369189 7870209.43
0.285268327 4553364.04
0.450007519 8502200.81
0.430246027 8350119.19
0.95087211 3309957.68
0.356794046 2488728.17
0.497053138 4178706.79
0.563165348 8852929.5
0.967759368 6855002.44
0.567805379 1823687.59
0.203961873 296646.427
0.547594383 1577649.28
0.211286623 2039906.33
0.193097376 672940.43
0.782049467 9015215.26
0.422007772 3025898.54
0.797120597 9895254.26
0.181940283 3874716.07
0.737382928 2247313.94
0.664735985 5335070.71
0.468823843 2018309.75
0.0974637872 5664972.39
0.0173116657 2770236.92
0.886939683 5638840.72
0.0620209302 701900.742
0.30706457 2533067.66
0.134073245 8966398.56
