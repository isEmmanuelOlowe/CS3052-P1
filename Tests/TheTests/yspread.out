52.0101032
